tests/pytests/unit/utils/test_nacl.py

```diff
@@ -1,10 +1,45 @@
 """
     Unit tests for the salt.utils.nacl module
 """
+import os
 
-import salt.utils.nacl
+import pytest
+
+import salt.modules.config as config
+import salt.utils.files
 from tests.support.mock import patch
 
+try:
+    import salt.utils.nacl as nacl
+
+    HAS_PYNACL = nacl.check_requirements
+except (ImportError, OSError, AttributeError):
+    HAS_PYNACL = False
+
+
+@pytest.mark.skipif(
+    not HAS_PYNACL, reason="skipping test_nacl, reason=PyNaCl is unavailable"
+)
```

Comment:
```suggestion
import salt.utils.nacl as nacl
pytest.importorskip("nacl")
```

Reply:
Rejecting, the test code works and playing with importorskip has introduced errors, hence stay with working code. Need to get this in, so can move on to other work. Will use importorskip in future work when creating new test files.